src/Containers/MainContainer/MainContainer.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+    Created by:                Level Up team
+    Date submitted (v4):       12/08/2020
+    File:                      MainContainer.js
+    File Description:          This component Contains the text and Button for the Table Management homepage.
+*/
+import React from 'react';
+import styled from 'styled-components';
+
+/**
+ * @returns {JSX.Element} Main container for the Demo page That contains
+ * the text and Button for the Table Management homepage.
+ */
+export interface IMainContainer {
+
+}
+
+export const MainContainer: React.FC<IMainContainer> = () => {
+    return (
+        <Pt6GreyBackground>
```

Comment:
PT PB PY are not clear variable names either describe them in a comment or replace with a better name. Ideally a better name